docs/gitlab-pipelines.md

```diff
@@ -0,0 +1,249 @@
+---
+id: gitlab-pipelines
+title: Testrunner Toolkit with GitLab Pipelines
+sidebar_label: GitLab Pipelines
+---
+
+The examples on this page can be applied to virtually any deployment provided that you already have some existing automated tests, and are either the maintainer or an admin of the target repository. 
+
+Your permissions must include:
+
+* ability to create and manage Gitlab repositories
+* ability to create and store [GitLab Environment Variables](https://docs.gitlab.com/ee/ci/variables/README.html)
+
+## What You'll Need
+
+* [GitLab Account](https://gitlab.com/users/sign_in)
+* [Sauce Labs Account](https://saucelabs.com/sign-up)
+
+## Set GitLab CI/CD Variables
+
+Set your Sauce Labs account credentials as [CI/CD environment variables](https://docs.gitlab.com/ee/security/cicd_environment_variables.html) by navigating to __Settings > CI/CD__:
+
+   ![GitLab Settings](assets/gitlab-settings.png)
+
+Then select __Variables__, followed by the __Add Variable__ button:
+
+   ![GitLab Variables](assets/gitlab-variables.png)
+   
+When you create the [CI/CD configuration file](#create-the-gitlab-configuration), your Sauce Labs authentication variables appear like so:
+
+```yaml
+variables:
+  SAUCE_USERNAME: ${SAUCE_USERNAME}
+  SAUCE_ACCESS_KEY: ${SAUCE_ACCESS_KEY}
+```
+
+## Create the Saucectl Configuration
+
+Create the `.sauce` directory at the root of your project and add a `config.yaml` file that points [`saucectl`](cli-reference.md) to your existing `tests` directory. 
```

Comment:
Saucectl ends up looking for `config.yml` by default. Reminds me of how I hate that both yml and yaml are acceptable :D
```suggestion
Create the `.sauce` directory at the root of your project and add a `config.yml` file that points [`saucectl`](cli-reference.md) to your existing `tests` directory. 
```